File: langs/lib_swift/primes.swift
// The Lumen library file langs/lib_lumen/primes.lm, ported by scripts/port_examples.py; edit the Lumen original, not this file.

func next_prime(n: Int) {
    var k = n + 1
    while true {
        if is_prime(n: k) {
            k
        }
        k = k + 1
    }
}

File: langs/lib_rust/primes.rs
// The Lumen library file langs/lib_lumen/primes.lm, ported by scripts/port_examples.py; edit the Lumen original, not this file.

fn next_prime(n: i64) {
    let mut k = n + 1;
    while true {
        if is_prime(k) {
            k;
        }
        k = k + 1;
    }
}
